### PR TITLE
Added symmetric tree algorithm in all four languages

### PR DESCRIPTION
**Pull request by atul1510**
## ✔️ Check List 

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## 📄 Note to reviewers

Problem Description:
Given a binary tree, check whether it is a mirror of itself.

Examples:
1] INPUT:
1
/
2 2
/ \ /
3 4 4 3

OUTPUT: Symmetric

2] INPUT:
1
/
2 2
\
3 3

OUTPUT: Not symmetric

## 📷 Screenshots
Screenshot (178)

### CODE REVIEW

**Review by Arun9739 — CHANGES_REQUESTED** [review]
Take input from user, instead of giving directly in program

**Review by 072arushi — APPROVED** [review]

**Review by zugzwang03 — APPROVED** [review]

**Review by siddhi-244 — APPROVED** [review]

**Review by Sanjanabharadwaj25 — APPROVED** [review]

**Review by Kumar-laxmi — CHANGES_REQUESTED** [review]
Change file ```Node.java``` to ```Symmetric_Tree.java```

**Review by 072arushi — APPROVED** [review]

**Comment by atul1510** [comment]
> Take input from user, instead of giving directly in program

@Arun9739 Done with all the changes requested, Kindly review it.

**Comment by atul1510** [comment]
> Change file `Node.java` to `Symmetric_Tree.java`

@Kumar-laxmi sir, have done with all the changes requested, kindly review it .

File: Java/Trees/Node.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+
```
Comment (Kumar-laxmi) [inline Java/Trees/Node.java:1]: Why have you included this file

Comment (atul1510) [reply Java/Trees/Node.java]: @Kumar-laxmi sir, Done with all the changes,

**Review by Kumar-laxmi — APPROVED** [review]

**Pull request merged**